src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -94,14 +91,11 @@ public CommandResult execute(Model model) throws CommandException {
      */
     private static Person createEditedPerson(Person personToEdit, EditPersonDescriptor editPersonDescriptor) {
         assert personToEdit != null;
-
-        //        Name updatedName = editPersonDescriptor.getName().orElse(personToEdit.getName());
-        //        Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
-        //        Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
-        //        Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
-        //        Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
-        //        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags);
-        return null;
+        Name updatedName = editPersonDescriptor.getName().orElse(personToEdit.getName());
+        Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
+        Nric updatedNric = editPersonDescriptor.getNric().orElse(personToEdit.getNric());
+        DoB updatedDob = editPersonDescriptor.getDob().orElse(personToEdit.getDoB());
+        return new Patient(updatedNric, updatedName, updatedDob, updatedPhone);
```

Comment:
I think this one can return a doctor too, so might wanna rethink the return statement